Sync is failing

```go
go build -o build/its main.go
./build/its sync
Error: error creating syncer: failure hydrating trakt client: failure generating auth codes: http request POST https://api.trakt.tv/oauth/device/code returned status code 401: unexpected status code 401
make: *** [Makefile:7: sync] Error 1
Error: Process completed with exit code 2.
```

I see it in your actions log too

--- a/pkg/client/trakt.go
+++ b/pkg/client/trakt.go
@@ -250,7 +250,11 @@ func (tc *TraktClient) GetAuthCodes() (*entities.TraktAuthCodesResponse, error) 
 		BasePath: traktPathBaseAPI,
 		Endpoint: traktPathAuthCodes,
 		Body:     bytes.NewReader(body),
-		Headers:  tc.defaultApiHeaders(),
+		Headers: map[string]string{
+			traktHeaderKeyApiVersion:  "2",
+			traktHeaderKeyContentType: "application/json",
+			traktHeaderKeyApiKey:      *tc.config.ClientID,
+		},
 	})
 	if err != nil {
 		return nil, err

